### PR TITLE
Convert to valid HTML 5

### LINKED ISSUE #8: Convert MathML to not using namespaces so that it can be parsed by HTML.
The alternative as per http://stackoverflow.com/questions/25952401/declaring-xml-namespaces-for-mathml/25968000?iemail=1&noredirect=1#25968000 is to convert the entire site into xhtml.

### PR DESCRIPTION
**Pull request by reima**
# tl;dr

All generated HTML pages are now valid HTML 5, with 0 errors and 0 warnings on the [W3C Nu Html Checker](https://validator.w3.org/nu/about.html).

# Motivation

Running [docs.gl](http://docs.gl) through an [HTML validator](https://validator.w3.org/) yields tens to hundreds of errors per page. While most browsers are resilient against such errors, having standards conforming markup helps a long way to ensure the website is displayed correctly in all browsers.

Many of these errors could be resolved automatically by declaring the website as HTML 5 (currently it's declared as XHTML 1.0 Transitional), because it already uses HTML 5 features (like `<meta charset="utf-8">`). Besides, switching to HTML 5 has some additional benefits:

* It's the current standard and as such the most future proof
* HTML 5 promotes semantic markup (separation of content from presentation)
* MathML can be embedded without the unwieldly mml namespace (fixes #8)

Possible downsides from switching to HTML 5:

* We further diverge from the original OpenGL manpages

# Approach

* Switch the DOCTYPE to HTML 5
* Run all pages through (an offline version of) the [W3C Nu Html Checker](https://validator.w3.org/nu)
* Fix all the errors:
    * Replace obsolete attributes with equivalent CSS
    * Remove mml namespace from MathML
    * Fix invalid MathML
    * Fix stray/invalid tags
    * Fix unescaped < or > characters in source code examples
    * Fix broken table rowspans

Only the first two types of errors were introduced by switching to HTML 5. All the other errors were already there before and could be revealed systematically with the Html Checker.

Many changes were made by bulk search & replace operations over all files. While I haven't manually inspected every change, I'm confident I didn't break anything in the process.

# Result

All pages are now valid HTML 5 according to [W3C Nu Html Checker](https://validator.w3.org/nu). None of the content was changed, only its markup. No effort was made to make the markup more semantic, but this could be something for a later PR.

In the process, some display errors were discovered and fixed, e.g.:

* MathML errors on [es2/glCompressedTexSubImage2D](http://docs.gl/es2/glCompressedTexSubImage2D), [es2/glCopyTexSubImage2D](http://docs.gl/es2/glCopyTexSubImage2D), [es2/glTexSubImage2D](http://docs.gl/es2/glTexSubImage2D), [gl3/glSamplerParameter](http://docs.gl/gl3/glSamplerParameter), [gl3/glTexParameter](http://docs.gl/gl3/glTexParameter) and others
* Broken page layout on [gl3/glFenceSync](http://docs.gl/gl3/glFenceSync), [gl3/glGetSync](http://docs.gl/gl3/glGetSync), [gl3/glWaitSync](http://docs.gl/gl3/glWaitSync)
* Invalid table rowspans on [gl3/glTexBuffer](http://docs.gl/gl3/glTexBuffer)

Many, many other errors had no visible effect (because the browsers can cope with them), but were fixed nonetheless.

### CODE REVIEW

**Comment by BSVino** [comment]
This is wow, great, and yes I want it, but woo you caught me at a very busy time (I mean they're all busy times but this time is especially busy) so you may have to give me a week or two before I can give it a serious inspection. Sorry about the delay.

**Comment by reima** [comment]
No pressure, take your time. Thanks for the quick response and the heads-up!

**Comment by BSVino** [comment]
OK I checked it all out. I don't see anything very scary, but it should probably be tested on a few different browsers before I push it up. It works fine on my Safari, have you tested it on anything else?

**Comment by reima** [comment]
Thanks for checking it out. I mostly used Chrome and some Firefox while working on this. It looked fine in both.

**Comment by BSVino** [comment]
Works for me then. Thanks again, this is great work.
